### DISCUSSION TITLE
Not able to find out 'How to use Vue router' when using @astrojs/vue

### DISCUSSION THREAD

**Discussion by amitleuva1987** [withastro/roadmap#331] [Proposal]
Hi,

I am using vue components in Astro. 

```
import { defineConfig } from 'astro/config';

import vue from "@astrojs/vue";

// https://astro.build/config
export default defineConfig({
  integrations: [vue()]
});
```

I have installed few modules. But, I am not able to find out any documentation on how to Integrate "Vue Router"  with Astro. 

I can see that I am able to use "<a>" anchor to navigate to pages in vue components, But, its refreshing the whole page.  

Also, It would be helpful if there is any other way to navigate pages.

**Comment by panwauu** [comment]
AFAIK client side routing is currently not supported as Astro is a MPA focused framework. From the docs: "Astro is an MPA framework"

Also you will get the best support at the discord server: https://discord.com/channels/830184174198718474/1019713903481081876
